core/embed/projects/unix/main.c

```diff
@@ -568,7 +568,7 @@ MP_NOINLINE int main_(int argc, char **argv) {
 #ifdef MICROPY_PY_SYS_PATH_DEFAULT
     path = MICROPY_PY_SYS_PATH_DEFAULT;
 #else
-    path = ".frozen:~/.micropython/lib:/usr/lib/micropython";
+    path = ".frozen:~/.micropython/lib";
```

Comment:
`~/.micropython/lib` also seems questionable, if nobody is using it I'd be in favor of dropping it too

Reply:
2975a07f1